lib/util/print.js

```diff
@@ -106,4 +107,46 @@ function printTreeTraversal(node, parentNode) {
   }
 }
 
+// Prints an expression node in LaTeX
+function printLatex(node, showPlusMinus=false) {
+  // Creates the LaTeX
+  function intoTex(node, showPlusMinus=false) {
+    const originalNodeTex = node.toTex();
+
+    let outputNodeTex;
+
+    if (!showPlusMinus) {
+      // Replaces '+-' with '-'
```

Comment:
can you add the `// (The + - is needed to support the conversion of subtraction to addition of....` comment from above here too? it's a weird thing and I want people looking at this code to know why this is happening